### ISSUE TITLE
Template generation on codethesaur.us is not reachable anymore

### ISSUE THREAD

**Issue by cafce25** [codethesaurus/codethesaur.us#605]
# Expected Behavior

I can select language, version and structure triples that are not available yet as reference to generate a template.
The button should change from `Go ->` to `Generate Template`

# Actual Behavior

When a structure is selected all (language, version) tuples that do not provide info for the structure are disabled and cannot be selected any more.

# Steps to Reproduce the Problem

1. Go to https://codethesaur.us 
1. Select a structure
1. Unavailable structures are disabled

# Additional Info

As Nekros1712 noted in #544: #496 broke how templates can be generated on the site.

The expected behavior is my imagination of a solution, anything else that allows easy generation of templates on codethesaur.us is fine.

**Comment by cafce25** [comment]
As a workaround for now you can visit `https://codethesaur.us/reference/?lang=<language>%3B<version>&concept=<structure>` to still generate a template or use `python manage.py generate_templates --language_version=<language_version> <language> <structure>` from within the docker container

**Comment by geekygirlsarah** [comment]
Yeah, I meant to file this yesterday as a friend realized they didnt' have a good way to make a new one. I'm wondering if something like a dropdown of concept and language either in the blue box at the bottom of the home page, or something on the Contribute page could maybe show the previous generated template that used to be there?
